demisto_sdk/commands/common/hook_validations/release_notes.py

```diff
@@ -36,6 +42,140 @@ def __init__(self, release_notes_file_path, modified_files=None, pack_name=None,
+                    else:
+                        headers[content_type] = [header]
+        return headers
+
+    def validate_special_forms(self, headers: Dict) -> bool:
+        is_valid = True
+        for content_type, content_items in headers.items():
+            if not content_items:
+                error_message, error_code = Errors.release_notes_invalid_content_type_header(content_type=content_type,
+                                                                                             pack_name=self.pack_name)
+                if self.handle_error(error_message, error_code, file_path=self.release_notes_file_path,
+                                     drop_line=True):
+                    is_valid = False
+            if content_type.lower().replace(' ', '') in RN_CONTENT_ENTITY_WITH_STARS:
+                for content_item in content_items:
+                    if content_item.count('**') != 2:
+                        error_message, error_code = Errors.release_notes_invalid_content_type_header(
+                            content_type=content_type,
+                            pack_name=self.pack_name)
+                        if self.handle_error(error_message, error_code, file_path=self.release_notes_file_path,
+                                             drop_line=True):
+                            is_valid = False
+        return is_valid
+
+    def filter_rn_headers(self, headers: Dict) -> None:
+        """
+            Filters out the headers from the release notes file, removing add-ons such as "New" and "**".
+        Args:
+            headers: (Dict) - The release notes headers to filter, the structure is content type -> headers.(e.g. Integrations -> [header1, header2])
+        Return:
+            None.
+        """
+        for content_type, content_items in headers.items():
+            content_items = self.filter_nones(ls=content_items)
+            if content_type in RN_CONTENT_ENTITY_WITH_STARS and content_items:
+                headers[content_type] = [item.replace('*', '') for item in content_items]
+            if content_type.lower().replace(' ', '') in RN_CONTENT_ENTITY_WITH_STARS:
+                content_items = list(map(lambda x: x.replace('**', ''), content_items))
+            headers[content_type] = [item.replace('New:', '').strip() for item in content_items]
+
+    def validate_content_type_header(self, content_type: str) -> bool:
+        """
+            Validate that the release notes 1st headers (the content type) are a valid content entity.
+        Args:
+            content_type: (str) - The content type to validate.(e.g. Integrations, Playbooks, etc.)
+        Return:
+            True if the content type is valid, False otherwise.
+        """
+        # Get all the content type headers
+        rn_valid_headers = list(map(lambda rn_valid_header: rn_valid_header.lower(), RN_HEADER_BY_FILE_TYPE.values()))
+
+        if content_type.lower() not in rn_valid_headers:
```

Comment:
```suggestion
        if content_type not in rn_valid_headers:
```
needs to be case sensitive